openwisp_utils/admin_theme/templatetags/ow_tags.py

```diff
@@ -21,4 +21,7 @@ def ow_create_filter(cl, spec, total_filters):
 @register.filter
 @stringfilter
 def join_string(value):
+    '''
+    Can be used to join strings with "-" to make id or class
+    '''
```

Comment:
pelase use double quotes for docstrings, `""""` for consistency with the rest of the codebase

Reply:
I did this again. very bad